src/js/dp/search.js

```diff
@@ -34,13 +45,28 @@ if (searchContainer) {
       );
     }
 
+    // if it takes more than 500ms to retreive results, show a loading message
+    var loadingPanel = setTimeout(() => {
+      document.querySelector('#results-loading').classList.remove('hide')
```

Comment:
Think this could benefit from being set to a `const` and a guard clause added
```suggestion
      const resultsLoader = document.querySelector('#results-loading')
      if (resultsLoader) {
         resultsLoader.classList.remove('hide')
      }
```